float32[15] to_degree
uint32 to_msec
bool wait
---
bool success
